browser.js

```diff
@@ -0,0 +1,2 @@
+window.CodeMirror = require("codemirror");
```

Comment:
I am not sure you need to do this even. Just add the `const` in the other file and you should be good-to-go. 

Here is another reference example: https://github.com/graphql/graphiql/blob/master/packages/codemirror-graphql/src/mode.js

Reply:
I had to add this to make sure the author's examples continued to work. The author uses the lib with browserify. So I moved out the pieces that was needed for the demo into this file. You can see that `index.html` (demo) uses `solidity.js` file.